### PR TITLE
Auto-expand all messages and schemas

### PR DESCRIPTION
**Pull request by aaronforinton-dojo**
Auto-expanding all messages and schemas per convo with @ryanlaycock @zianyusuf 

We want the API docs to have their payloads and schemas auto-expanded by default.

### CODE REVIEW

File: library/src/components/Schema.tsx [hunk 1]
```diff
@@ -26,11 +26,11 @@ export const Schema: React.FunctionComponent<Props> = ({
   isProperty = false,
   isCircular = false,
   dependentRequired,
-  expanded = false,
+  expanded = true,
   onlyTitle = false,
 }) => {
   const { reverse } = useContext(SchemaContext);
-  const [expand, setExpand] = useState(expanded);
+  const [expand, setExpand] = useState(true);
```
Comment (jackhopner) [inline library/src/components/Schema.tsx]: why this change, if expanded is set to true by default then we should be able to use that var?

**Comment by aaronforinton-dojo** [comment]
@mofesal-dojo I'm trying to have everything expanded by default when a user opens the page. Do you think this will do it? Conscious of the fact it's hard to preview these changes

**Comment by mofesal-dojo** [comment]
@aaronforinton-dojo Yeah this looks like it makes sense! I guess locally, you could also try to build the docs of this branch and just double check it has the desired result?
